packages/helm/symphony/files/symphony-api.json

```diff
@@ -474,6 +486,12 @@
                 "config": {}
                 {{- end }}
               },
+              "mem-keylock": {
```

Comment:
If I remember correctly, we agree we need to let the manager keep its key lock provider mode in code, instead of passing the configuration. Will you follow up this on next PR?

Reply:
Yes, but I found we cannot let the manager keep its key lock provider mode, or the unit test will fail...

Reply:
Can you elaborate on this, @FireDefend?

Reply:
In the current design, we need to create a global mode keylock provider first before we create or use any shared mode keylock provider. But in the unit test, it cannot meet the prerequisite (all the unit tests are parallelled, they don't have a global mode keylock). 

So, solution manager will use dedicated keylock (unit test) or shared keylock (production code). I cannot hardcode this part inside solution manager, this will be decided by config.

Reply:
@Haishi2016 
Hi Haishi, do you have other comments? We would like to check in this PR since other features may also depend on this PR.